resspect/fit_lightcurves.py

```diff
@@ -15,16 +15,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from resspect.bazin import bazin, fit_scipy
-from resspect.exposure_time_calculator import ExpTimeCalc
-from resspect.snana_fits_to_pd import read_fits
-
+import os
 import io
+import tarfile
+from typing import Tuple
```

Comment:
isort should suggest a different import ordering

Reply:
Locally I am checking import ordering with `pylint` and the above ordering looks fine with it. Since we have not fixed a code analysis tool yet, I would continue with  `pylint` import guidelines